### ISSUE TITLE
Add option to hide "requested" media

### ISSUE THREAD

**Issue by TheOnlyYogo** [sct/overseerr#3681]
### Description

In the Discover section, there is already an option to hide "Available" media in the settings. However there is not option to hide "Requested" media/titles.

### Desired Behavior

Add an option in settings to hide already requested media/titles from the Discover section.

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by Ektokid** [comment]
Can we please get this merged, I would love this feature so much!

**Comment by TheOnlyYogo** [comment]
I'd hate for this to be closed before merge. Are there any roadblocks for this to be implemented?

**Comment by yourfavoriteweapon** [comment]
this is a much needed feature, im shocked it isn't already an option, should probably be the default behavior.

**Comment by siggijarl** [comment]
I need this.

**Comment by Manfred73** [comment]
When will this become available?

**Comment by Ektokid** [comment]
Need this feature.

**Comment by TheOnlyYogo** [comment]
I'm awful with code... but as there is already an option to hide "available" media, wouldn't the solution to this feature request be to just copy the code and replace "available" with "requested" on some variable or value?

**Comment by R0cketM0nster** [comment]
Plus one requesting this feature

**Comment by HillbillyRockHillbillyRoll** [comment]
Yeah, this would make light work when "discovering" content to add, having the ability to hide requested content would just make that a bit more seamless.

**Comment by HillbillyRockHillbillyRoll** [comment]
Small little script to help others right now till this is released, loaded up using tampermonkey, just replace your domain in the script settings to match the page you want to monitor - https://greasyfork.org/en/scripts/521121-overseer-hide-requested-items

**Comment by yourfavoriteweapon** [comment]
What is blocking this from moving forward?

**Comment by yourfavoriteweapon** [comment]
Any update?

**Comment by TheOnlyYogo** [comment]
> Any update?

Nope ... I think this will disappear into the ether.... it's a shame because I think it would be very useful...

**Comment by HillbillyRockHillbillyRoll** [comment]
You are better off opening an issue on Jellyseerr IMO (https://github.com/fallenbagel/jellyseerr)

**Comment by siggijarl** [comment]
> Small little script to help others right now till this is released, loaded up using tampermonkey, just replace your domain in the script settings to match the page you want to monitor - https://greasyfork.org/en/scripts/521121-overseer-hide-requested-items

thanks man, this is a gamechanger

**Comment by Ektokid** [comment]
How has this not been implemented yet? So many requests. I'm about to run how to code this just to fix it. Using the workaround for now.

**Comment by nupulu** [comment]
Can we haz this palease?

**Comment by leitunes** [comment]
Up! This is a must!

**Comment by Electron707** [comment]
> Small little script to help others right now till this is released, loaded up using tampermonkey, just replace your domain in the script settings to match the page you want to monitor - https://greasyfork.org/en/scripts/521121-overseer-hide-requested-items

This is excellent!
